tests/scenarios/k8s/ipoib/base/ipoib.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: placeholder
+spec:
+  template:
+    metadata:
+      annotations:
+        # This name should match the IPoIBNetwork object we created earlier.
+        # You can find this config by running `kubectl get IPoIBNetwork`.
+        k8s.v1.cni.cncf.io/networks: aks-infiniband
```

Comment:
Any plans to test all 8 NICs?

Reply:
I don't know if we can control the mapping of the IP inside the pod and the actual physical interface behind it. It is kind of abstracted away from the user.